LondonBritishLibrary/orient/BLorient672.xml

```diff
@@ -241,7 +268,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <layout columns="3" writtenLines="31 33">
                                     <ab type="pricking">Pricking and ruling are visible.</ab>
                                     <ab type="pricking">Primary pricks and horizontal pricks are visible.</ab>
-                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <!--vertical lines stop at the pricks, is it still A?-->
```

Comment:
this is a question for @DenisNosnitsin1970 I guess )

Reply:
discussed with him orally